var/spack/repos/builtin/packages/hip/package.py

```diff
@@ -389,6 +389,9 @@ def get_paths(self):
                 "rocm-device-libs": self.spec["llvm-amdgpu"].prefix,
             }
 
+        if self.spec.satisfies("@5.4:"):
+            paths["hipify-clang"] = rocm_prefix
```

Comment:
This really needs to be split between the external case and the not external case. In the external case it's in `rocm_prefix`, but in the spack-built case it is in `self.spec["hipify-clang"].prefix`

Reply:
OK. Just pushed update.